test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -655,8 +651,8 @@ TEST_F(AmetsuchiTest, FindTxByHashTest) {
 
   // TODO: 31.10.2017 luckychess move tx3hash case into a separate test after
   // ametsuchi_test redesign
-  auto tx1hash = txn1.hash();
-  auto tx2hash = txn2.hash();
+  auto tx1hash = txs[0].hash();
```

Comment:
It is probably more safe to use `.at()` since it will throw if someone will modify test without noticing.